examples/angular2/services/store.js

```diff
@@ -0,0 +1,75 @@
+var Todo = (function () {
```

Comment:
0) Any reason why this is not in TS?
1) Is this still processed somehow? Can we use `const` here?